Example/Tests/Tests.m

```diff
@@ -423,21 +461,28 @@ - (ADBMediaHeartbeat *)createWithDelegate:(id)delegate andConfig:(ADBMediaHeartb
             });
 
             it(@"Video Playback Started initializes ADBMediaHeartbeat object with default values", ^{
-                mockADBMediaHeartbeat = mock([ADBMediaHeartbeat class]);
                 SEGMockADBMediaHeartbeatFactory *mockADBMediaHeartbeatFactory = [[SEGMockADBMediaHeartbeatFactory alloc] init];
                 mockADBMediaHeartbeatFactory.ADBMediaHeartbeat = mockADBMediaHeartbeat;
+
+                SEGMockADBMediaObjectFactory *mockADBMediaObjectFactory = [[SEGMockADBMediaObjectFactory alloc] init];
```

Comment:
Not a blocker but it's better to have the initializer accept the media object rather than setting it separately. Less of a chance that the factory is in an invalid state.